### ISSUE TITLE
com.facebook.jni.CppException: Cannot read property 'value' of undefined

### ISSUE THREAD

**Issue by denisk20** [software-mansion/react-native-reanimated#4417]
### Description

After updating to 3.1.0 I'm seeing Android crashes in our crashlytics:
```
Fatal Exception: com.facebook.jni.CppException: Cannot read property 'value' of undefined

TypeError: Cannot read property 'value' of undefined
    at anonymous (JavaScript:1:306)
    at apply (native)
    at anonymous (JavaScript:1:111)
    at forEach (native)
    at anonymous (JavaScript:1:78)
       at com.swmansion.reanimated.Scheduler.triggerUI(Scheduler.java)
       at com.swmansion.reanimated.Scheduler$1.run(Scheduler.java:24)
       at com.swmansion.reanimated.Scheduler$2.runGuarded(Scheduler.java:43)
       at com.facebook.react.bridge.GuardedRunnable.run(GuardedRunnable.java:30)
       at android.os.Handler.handleCallback(Handler.java:938)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loopOnce(Looper.java:201)
       at android.os.Looper.loop(Looper.java:288)
       at android.app.ActivityThread.main(ActivityThread.java:7870)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
```
It doesn't seem to be specific to any android version / device (I've seen the reports from android 10, 12).

### Steps to reproduce

Can't tell any more than the stacktrace above.

### Snack or a link to a repository

n/a

### Reanimated version

3.1.0

### React Native version

0.71.7

### Platforms

Android

### JavaScript runtime

Hermes

### Workflow

None

### Architecture

None

### Build type

None

### Device

None

### Acknowledgements

Yes

**Comment by Latropos** [comment]
@denisk20 I've been looking for similar issues to get more info about it and found none. Therefore I suppose that there is something wrong on your side. If you don't provide any repro we will close the issue since we will be unable to help you anyway.

**Comment by vitorsilvalima** [comment]
I'm also getting the same error in productions

```
Fatal Exception: com.facebook.jni.CppException: Trying to access property `dismiss` of an object which cannot be sent to the UI runtime.

Error: Trying to access property `dismiss` of an object which cannot be sent to the UI runtime.
    at get (JavaScript:1:132)
    at handleOnEnd (JavaScript:1:1290)
    at anonymous (JavaScript:1:248)
    at anonymous (JavaScript:1:384)
    at handleAndFlushAnimationFrame (JavaScript:1:147)
       at com.swmansion.reanimated.nativeProxy.EventHandler.receiveEvent(EventHandler.java)
       at com.swmansion.reanimated.nativeProxy.EventHandler.receiveEvent(EventHandler.java:25)
       at com.swmansion.gesturehandler.react.RNGestureHandlerStateChangeEvent.dispatch(RNGestureHandlerStateChangeEvent.kt:42)
       at com.swmansion.reanimated.NodesManager.handleEvent(NodesManager.java:309)
       at com.swmansion.reanimated.NodesManager.onEventDispatch(NodesManager.java:285)
       at com.facebook.react.uimanager.events.EventDispatcherImpl.dispatchEvent(EventDispatcherImpl.java:116)
       at com.swmansion.gesturehandler.ReactContextExtensionsKt.dispatchEvent(ReactContextExtensions.kt:9)
       at com.swmansion.gesturehandler.react.RNGestureHandlerModule.sendEventForDirectEvent(RNGestureHandlerModule.kt:634)
       at com.swmansion.gesturehandler.react.RNGestureHandlerModule.sendEventForReanimated(RNGestureHandlerModule.kt:620)
       at com.swmansion.gesturehandler.react.RNGestureHandlerModule.onStateChange(RNGestureHandlerModule.kt:572)
       at com.swmansion.gesturehandler.react.RNGestureHandlerModule.access$onStateChange(RNGestureHandlerModule.kt:38)
       at com.swmansion.gesturehandler.react.RNGestureHandlerModule$eventListener$1.onStateChange(RNGestureHandlerModule.kt:346)
       at com.swmansion.gesturehandler.core.GestureHandler.dispatchStateChange(GestureHandler.kt:84)
       at com.swmansion.gesturehandler.core.GestureHandlerOrchestrator.onHandlerStateChange(GestureHandlerOrchestrator.kt:155)
       at com.swmansion.gesturehandler.core.GestureHandler.moveToState(GestureHandler.kt:551)
       at com.swmansion.gesturehandler.core.GestureHandler.end(GestureHandler.kt:666)
       at com.swmansion.gesturehandler.core.PanGestureHandler.onHandle(PanGestureHandler.kt:251)
       at com.swmansion.gesturehandler.core.GestureHandler.handle(GestureHandler.kt:374)
       at com.swmansion.gesturehandler.core.GestureHandlerOrchestrator.deliverEventToGestureHandler(GestureHandlerOrchestrator.kt:274)
       at com.swmansion.gesturehandler.core.GestureHandlerOrchestrator.deliverEventToGestureHandlers(GestureHandlerOrchestrator.kt:229)
       at com.swmansion.gesturehandler.core.GestureHandlerOrchestrator.onTouchEvent(GestureHandlerOrchestrator.kt:45)
       at com.swmansion.gesturehandler.react.RNGestureHandlerRootHelper.dispatchTouchEvent(RNGestureHandlerRootHelper.kt:97)
       at com.swmansion.gesturehandler.react.RNGestureHandlerRootView.dispatchTouchEvent(RNGestureHandlerRootView.kt:35)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3083)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2741)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3083)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2741)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3083)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2741)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3083)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2741)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3083)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2741)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3083)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2741)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3083)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2741)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3083)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2741)
       at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:441)
       at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1830)
       at android.app.Activity.dispatchTouchEvent(Activity.java:3400)
       at androidx.appcompat.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:69)
       at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:399)
       at android.view.View.dispatchPointerEvent(View.java:12789)
       at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:5288)
       at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5082)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4595)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4648)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4614)
       at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4754)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4622)
       at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4811)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4595)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4648)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4614)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4622)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4595)
       at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:7351)
       at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:7320)
       at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:7271)
       at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:7468)
       at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:186)
       at android.os.MessageQueue.nativePollOnce(MessageQueue.java)
       at android.os.MessageQueue.next(MessageQueue.java:326)
       at android.os.Looper.loop(Looper.java:160)
       at android.app.ActivityThread.main(ActivityThread.java:6704)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:916)
```

**Comment by vitorsilvalima** [comment]
Android 9 and 13

**Comment by enchorb** [comment]
Not sure if this is related but seem to be seeing a similar error in our Crashlytics as well on 3.1.0

```
Fatal Exception: com.facebook.jni.CppException: Exception in HostFunction: Incompatible object passed to scheduleOnJS. It is only allowed to schedule functions defined on the React Native JS runtime this way.

Error: Exception in HostFunction: Incompatible object passed to scheduleOnJS. It is only allowed to schedule functions defined on the React Native JS runtime this way.
    at _scheduleOnJS (native)
    at anonymous (JavaScript:1:192)
    at anonymous (JavaScript:1:211)
    at anonymous (JavaScript:1:628)
    at handleAndFlushAnimationFrame (JavaScript:1:147)
       at com.swmansion.reanimated.nativeProxy.EventHandler.receiveEvent(SourceFile)
       at com.swmansion.reanimated.nativeProxy.EventHandler.receiveEvent(:21)
       at com.swmansion.gesturehandler.react.RNGestureHandlerStateChangeEvent.dispatch(RNGestureHandlerStateChangeEvent.java:13)
       at com.swmansion.reanimated.NodesManager.handleEvent(:16)
       at com.swmansion.reanimated.NodesManager.onEventDispatch(:6)
       at com.facebook.react.uimanager.events.EventDispatcherImpl.dispatchEvent(:27)
       at com.swmansion.gesturehandler.ReactContextExtensionsKt.dispatchEvent(ReactContextExtensions.kt:25)
       at com.swmansion.gesturehandler.react.RNGestureHandlerModule.sendEventForDirectEvent(:9)
       at com.swmansion.gesturehandler.react.RNGestureHandlerModule.sendEventForReanimated()
       at com.swmansion.gesturehandler.react.RNGestureHandlerModule.onStateChange(:24)
       at com.swmansion.gesturehandler.react.RNGestureHandlerModule.access$onStateChange()
       at com.swmansion.gesturehandler.react.RNGestureHandlerModule$eventListener$1.onStateChange(RNGestureHandlerModule.java:7)
       at com.swmansion.gesturehandler.core.GestureHandler.dispatchStateChange(GestureHandler.java:8)
       at com.swmansion.gesturehandler.core.GestureHandlerOrchestrator.onHandlerStateChange(GestureHandlerOrchestrator.java:93)
       at com.swmansion.gesturehandler.core.GestureHandler.moveToState(GestureHandler.java:45)
       at com.swmansion.gesturehandler.core.GestureHandler.cancel(GestureHandler.java:14)
       at com.swmansion.gesturehandler.core.GestureHandlerOrchestrator.cancelAll(GestureHandlerOrchestrator.java:47)
       at com.swmansion.gesturehandler.core.GestureHandlerOrchestrator.onTouchEvent(GestureHandlerOrchestrator.java:21)
       at com.swmansion.gesturehandler.react.RNGestureHandlerRootHelper.dispatchTouchEvent(RNGestureHandlerRootHelper.java:13)
       at com.swmansion.gesturehandler.react.RNGestureHandlerRootView.dispatchTouchEvent(RNGestureHandlerRootView.java:14)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3091)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3091)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3091)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3091)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3091)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3091)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3091)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3091)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
       at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:3091)
       at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2802)
       at com.android.internal.policy.DecorView.superDispatchTouchEvent(DecorView.java:500)
       at com.android.internal.policy.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1905)
       at android.app.Activity.dispatchTouchEvent(Activity.java:4298)
       at androidx.appcompat.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:2)
       at com.android.internal.policy.DecorView.dispatchTouchEvent(DecorView.java:458)
       at android.view.View.dispatchPointerEvent(View.java:15269)
       at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:6757)
       at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:6557)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6013)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6070)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6036)
       at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:6201)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6044)
       at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:6258)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6017)
       at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:6070)
       at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:6036)
       at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:6044)
       at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:6017)
       at android.view.ViewRootImpl.deliverInputEvent(ViewRootImpl.java:9103)
       at android.view.ViewRootImpl.doProcessInputEvents(ViewRootImpl.java:9054)
       at android.view.ViewRootImpl.enqueueInputEvent(ViewRootImpl.java:9023)
       at android.view.ViewRootImpl$WindowInputEventReceiver.onInputEvent(ViewRootImpl.java:9226)
       at android.view.InputEventReceiver.dispatchInputEvent(InputEventReceiver.java:267)
       at android.os.MessageQueue.nativePollOnce(MessageQueue.java)
       at android.os.MessageQueue.next(MessageQueue.java:335)
       at android.os.Looper.loopOnce(Looper.java:161)
       at android.os.Looper.loop(Looper.java:288)
       at android.app.ActivityThread.main(ActivityThread.java:7884)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
```

**Comment by erksch** [comment]
@denisk20 having the same problem, where you able to resolve it?

**Comment by denisk20** [comment]
@denisk20 I think the issue might have been resolved in 3.3.0, at least I don't see the errors in crashlytics.

**Comment by Latropos** [comment]
Closing since the issue is gone in newer versions 👍
